packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -83,6 +85,7 @@ class _CodeSizeBodyState extends State<CodeSizeBody>
   @override
   void initState() {
     super.initState();
+    screen(_id);
```

Comment:
All the other screens call `screen(ScreenClass.id)`. Why did we make code size a top level private field instead of being a static member on the class?

Reply:
That was my original approach and I thought I cleaned up all cases. Missed this one though. Will fix.